src/components/routes/TextWithVisual.tsx

```diff
@@ -0,0 +1,27 @@
+import { FunctionComponent, ReactNode } from 'react'
+
+interface Props {
+  id?: string
+  positionOfVisual: 'left' | 'right'
+  children: ReactNode
+  visual: ReactNode
+}
+
+const TextWithVisual: FunctionComponent<Props> = ({
+  id,
+  positionOfVisual,
+  children,
+  visual,
+}) => (
+  <section
+    className={`text-navy-700 section flex flex-col md:flex-row ${
+      positionOfVisual === 'left' ? 'md:flex-row-reverse' : ''
+    }`}
+    {...(id && { id })}
```

Comment:
**suggestion (non-blocking):** you can simply write `id={id}` here. If it's not provided, it'll be `undefined`, which is an acceptable value.

Reply:
Ok, thanks for the feedback! I didn't check but maybe the id even gets removed by some webpack magic when it's undefined?